src/cmd/scorecard.go

```diff
@@ -0,0 +1,123 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/creasty/defaults"
+	"github.com/opslevel/opslevel-go/v2023"
+
+	"github.com/opslevel/cli/common"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var createScorecardCmd = &cobra.Command{
+	Use:   "scorecard",
+	Short: "Create a scorecard",
+	Long:  "Create a scorecard",
+	Example: `
+cat << EOF | opslevel create scorecard -f -
+name: "new scorecard"
+description: "a newly created scorecard"
+owner:
+  alias: "platform"
+EOF`,
+	Run: func(cmd *cobra.Command, args []string) {
+		input, err := readScorecardInput()
+		cobra.CheckErr(err)
+		result, err := getClientGQL().CreateScorecard(*input)
+		cobra.CheckErr(err)
+		fmt.Printf("created scorecard: %s\n", result.Id)
+	},
+}
+
+var getScorecardCmd = &cobra.Command{
+	Use:        "scorecard ID|ALIAS",
+	Short:      "Get details about a scorecard",
+	Long:       "Get details about a scorecard",
+	Args:       cobra.ExactArgs(1),
+	ArgAliases: []string{"ID", "ALIAS"},
+	Run: func(cmd *cobra.Command, args []string) {
+		key := args[0]
+		scorecard, err := getClientGQL().GetScorecard(*opslevel.NewIdentifier(key))
+		cobra.CheckErr(err)
+		common.PrettyPrint(scorecard)
+	},
+}
+
+var listScorecardCmd = &cobra.Command{
+	Use:     "scorecard",
+	Aliases: []string{"scorecards"},
+	Short:   "List scorecards",
+	Long:    "List scorecards",
```

Comment:
The example i usually use for list is something that uses a jq expression to find a list of data - maybe like
```
opslevel list scorecards -o json | jq 'map(.Filter.Name)'
```